docs/concepts/work-pools.md

```diff
@@ -0,0 +1,274 @@
+| --api           | The API URL for the Prefect Orion server. Default is the value of `PREFECT_API_URL`.                                                       |
+| --run-once           | Only run agent polling once. By default, the agent runs forever. [default: no-run-once]                                                                          |
+| --prefetch-seconds   | The amount of time before a flow run's scheduled start time to begin submission. Default is the value of `PREFECT_AGENT_PREFETCH_SECONDS`. |
+| --hide-welcome       | Do not display the startup ASCII art for the agent process.                                                                                |               
+
+You must start an agent within an environment that can access or create the infrastructure needed to execute flow runs. Your agent will deploy flow runs to the infrastructure specified by the deployment.
+
+!!! tip "Prefect must be installed in execution environments"
+    Prefect must be installed in any environment in which you intend to run the agent or execute a flow run.
+
+!!! tip "`PREFECT_API_URL` setting for agents"
+    `PREFECT_API_URL` must be set for the environment in which your agent is running or specified when starting the agent with the `--api` flag. 
+
+    If you want an agent to communicate with Prefect Cloud or a Prefect Orion API server from a remote execution environment such as a VM or Docker container, you must configure `PREFECT_API_URL` in that environment.
+
+### Starting an agent
+
+Use the `prefect agent start` CLI command to start an agent. You must pass at least one work pool name or match string that the agent will poll for work. If the work pool does not exist, it will be created.
+
+<div class="terminal">
+
+```bash
+$ prefect agent start -p [work pool name]
+```
+
+</div>
+
+For example:
+
+<div class="terminal">
+
+```bash
+$ prefect agent start -p "my-pool"
+Starting agent with ephemeral API...
+  ___ ___ ___ ___ ___ ___ _____     _   ___ ___ _  _ _____
+ | _ \ _ \ __| __| __/ __|_   _|   /_\ / __| __| \| |_   _|
+ |  _/   / _|| _|| _| (__  | |    / _ \ (_ | _|| .` | | |
+ |_| |_|_\___|_| |___\___| |_|   /_/ \_\___|___|_|\_| |_|
+
+Agent started! Looking for work from queue(s): my-pool...
+08:31:57.197 | INFO    | prefect.agent - Created work pool 'my-pool'.
+```
+
+</div>
+
+In this case, Prefect automatically created a new `my-pool` work pool.
+
+By default, the agent polls the API specified by the `PREFECT_API_URL` environment variable. To configure the agent to poll from a different server location, use the `--api` flag, specifying the URL of the server.
+
+In addition, agents can match multiple work pools by providing a `--match` string instead of specifying all of the work pools. The agent will poll every work pool with a name that starts with the given string. New pools matching this prefix will be found by the agent without needing to restart it.
```

Comment:
In it's current implementation, `--match` matches on queues and not pools.
```suggestion
In addition, agents can match multiple queues in a work pool by providing a `--match` string instead of specifying all of the queues. The agent will poll every queue with a name that starts with the given string. New queues matching this prefix will be found by the agent without needing to restart it.
```